### PR TITLE
Add custom Egghead transformer

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Add custom Egghead transformer for [`gatsby-remark-embedder`](https://github.com/MichaelDeBoey/gatsby-remark-embedder)

### CODE REVIEW

**Review by kentcdodds — APPROVED** [review]
Splendid! Thank you 👍

File: embedder-transformers/egghead.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  return `https://${host}/${pathname}/embed${search}`
+}
+
+const getHTML = url => {
+  const iframeSrc = getEggheadIFrameSrc(url)
+
+  return `<iframe src="${iframeSrc}" style="width: 100%; height: 410px; border: none; border-radius: 4px;" allowfullscreen></iframe>`
+}
+
+export {getHTML, shouldTransform}
```
Comment (kentcdodds) [inline embedder-transformers/egghead.js]: I think this needs to be commonjs

**Comment by MichaelDeBoey** [comment]
@kentcdodds Fixed + all green 🙂
https://5dec43364845110008274bd8--kentcdodds.netlify.com/blog/inversion-of-control

**Comment by kentcdodds** [comment]
Thank you!

**Comment by MichaelDeBoey** [comment]
With pleasure 🙂

**Pull request merged**